lib/driver.js

```diff
@@ -256,15 +259,21 @@ class AndroidUiautomator2Driver extends BaseDriver {
       await this.initAUT();
     }
     //Adding AUT package name in the capabilities if package name not exist in caps
-    if (!this.caps.appPackage) {
-      this.caps.appPackage = appInfo.appPackage;
+    if (this.opts.appInfo && !this.caps.appPackage) {
```

Comment:
Thanks @jlipps, I missed this out... I will have a deeper look into that, meanwhile I will close the PR.